### PR TITLE
hw-mgmt: topology: Remove A2D i2c sensors from MSN2700/MSN2410 systems Remove A2D i2c sensors from MSN2700/MSN2410 systems management + CPU board

### PR DESCRIPTION
**Pull request by sholeksandr**
Signed-off-by: Oleksandr Shamray <oleksandrs@nvidia.com>

### CODE REVIEW

**Comment by sholeksandr** [comment]
RERUN

File: usr/usr/bin/hw-management.sh [hunk 1]
```diff
@@ -684,7 +711,7 @@ add_cpu_board_to_connection_table()
 					sku=$(< /sys/devices/virtual/dmi/id/product_sku)
 					case $sku in
 						HI116|HI112|HI124|HI100)
-							#Anaconda 100/200 Tigon Jaguar
+							#Anaconda 100/200 Tigon Jaguar removed A2D from BWD
```
Comment (vadimp-nvidia) [inline usr/usr/bin/hw-management.sh:714]: Add space after #
Replace internal system names (Anaconda, Tigon) with official.
# SN3700, ..  with no A2D device on COME module.

File: usr/usr/bin/hw-management.sh [hunk 2]
```diff
@@ -662,10 +680,19 @@ add_cpu_board_to_connection_table()
 
 	case $cpu_type in
 		$RNG_CPU|$IVB_CPU)
-			cpu_connection_table=( ${cpu_type0_connection_table[@]} )
-			;;
+			board=$(< /sys/devices/virtual/dmi/id/product_name)
+			case $board in
+				MSN241*|MSN27*)
+					# Spider Panther removed A2D from SFF
```
Comment (vadimp-nvidia) [inline usr/usr/bin/hw-management.sh:686]: Don't use internal names.
# SN2700, ... with no A2D device on SFF module.

**Comment by sholeksandr** [comment]
RERUN

**Comment by sholeksandr** [comment]
RERUN

**Comment by sholeksandr** [comment]
RERUN

**Comment by sholeksandr** [comment]
RERUN

**Pull request closed without merging**